docs/performance-best-practices.md

```
@@ -0,0 +1,343 @@
+
+// Less optimal: Switching between sheets
+BT.drawSprite(enemySheet, sprite1, pos1);
+BT.drawSprite(playerSheet, sprite2, pos2); // Batch break!
+BT.drawSprite(enemySheet, sprite3, pos3); // Batch break!
+```
+
+### Sprite Sheet Organization
+
+**Pack related sprites into single sheets:**
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)